lib/ansible/plugins/action/pause.py

```diff
@@ -44,7 +44,7 @@
     try:
         curses.setupterm()
         HAS_CURSES = True
-    except curses.error:
+    except (curses.error, TypeError):
```

Comment:
I think that is right but it could just as easily be `except Exception:`
(reason being that any failure there would mean that we don't have a working curses and need to resort to the fallback code)

Reply:
Just wondering why once was picked other the other